graphql/src/execution/execution.rs

```diff
@@ -199,11 +201,20 @@ lazy_static! {
         .expect("Invalid value for GRAPH_QUERY_CACHE_STALE_PERIOD environment variable")
     };
 
+    /// In how many shards (mutexes) the query block cache is split.
+    static ref QUERY_BLOCK_CACHE_SHARDS: u8 = {
+        std::env::var("GRAPH_QUERY_BLOCK_CACHE_SHARDS")
+        .unwrap_or("100".to_string())
```

Comment:
To get even distribution across the shards, maybe the default value should divide 256, i.e., either 64 or 128